app/assets/javascripts/constants/style_constants.es6.js

```diff
@@ -35,6 +35,14 @@ class StylesSingleton {
       },
     };
   }
+
+  get zIndex() {
+    return {
+      modal: 900,
+      overlay: 800,
+      dropdown: 700,
+    }
+  }
```

Comment:
Instead of this, I think something like the following would be better:
```
get zIndex() {
    return {
      one: 100,
      two: 200,
      three: 300,
      ...
      nine: 900,
    };
}
```
I've seen other companies such as Medium do this, and this will also be more useable if we have to put multiple components on the same "plane". Does that make sense?